NEWS.md

```diff
@@ -75,7 +86,7 @@ Note: We're only listing outstanding class updates.
 
 * ObjectSpace::WeakMap
 
-    * `ObjectSpace::WeakMap#delete` was added to eagerly clear weak map
```

Comment:
why do you remove backquote even though it is `.md`?

Reply:
It will be marked up as `<code>` without backquotes.

Reply:
@ko1 Without backticks RDoc is able to linkify the method name. 
The current version:
![image](https://github.com/ruby/ruby/assets/129656/1785a15f-158f-408e-8f74-956c02977928)
After the fix:
![image](https://github.com/ruby/ruby/assets/129656/7261cef8-5606-4b3b-952e-62fa5ca126ba)

Reply:
I see.

----

For me, it is better to allow backquotes and rdoc also accepts backquoted method names, because it is easy to copy & paste to other articles.

Reply:
You mean the case of "only method name enclosed in backquotes", right?
E.g., `` `ObjectSpace::WeakMap#delete` ``, but not `` `(ObjectSpace::WeakMap#delete)` ``.
It would be better but should be requested to ruby/rdoc, not here.